packages/preload-webview/src/index.ts

```diff
@@ -35,7 +35,7 @@ export const init = (): void => {
   webviewPreload.init().catch((error: unknown) => console.error('Error while initializing the exposure', error));
 };
 
-// do not call init methd in case of testing
-if (!process.env.VITEST) {
+// do not call init methd in case of testing, but call when running E2E tests, see #6740
+if (!process.env.VITEST || process.env.PLAYWRIGHT_E2E) {
```

Comment:
```suggestion
// do not call init method in case of testing
if (import.meta.env.MODE !== 'test') {
```

Reply:
use import MODE instead of VITEST

as for e2e we replace the mode when compiling to not be test